api-reference/docstubs/api/calendar-allowedcalendarsharingroles.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "allowedCalendarSharingRoles"
+description: ""
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# allowedCalendarSharingRoles
+
+Namespace: microsoft.graph
+
+
```

Comment:
Is there no default description? Even a **TODO Description**?  Same for YAML description.